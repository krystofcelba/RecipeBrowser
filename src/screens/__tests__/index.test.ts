import { Provider } from 'react-redux';
import { Navigation } from 'react-native-navigation';

import { registerScreens } from '../';

describe('registerScreens', () => {
  it('register components as expected', () => {
    Navigation.registerComponent = jest.fn();
    registerScreens({}, Provider);
    expect(Navigation.registerComponent.mock.calls).toMatchSnapshot();
  });
});
