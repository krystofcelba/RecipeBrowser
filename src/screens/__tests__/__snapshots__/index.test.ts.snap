// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`registerScreens register components as expected 1`] = `
Array [
  Array [
    "recipeBrowser.RecipesListTopBar",
    [Function],
    Object {},
    [Function],
  ],
  Array [
    "recipeBrowser.RecipesList",
    [Function],
    Object {},
    [Function],
  ],
  Array [
    "recipeBrowser.RecipeDetail",
    [Function],
    Object {},
    [Function],
  ],
  Array [
    "recipeBrowser.AddRecipeForm",
    [Function],
    Object {},
    [Function],
  ],
]
`;
